OpenRA.Game/Network/LocalizedMessage.cs

```diff
@@ -123,13 +114,18 @@ public string Translate(ModData modData)
 			var argumentDictionary = new Dictionary<string, object>();
 			foreach (var argument in Arguments)
 			{
-				if (argument.Type == FluentArgument.FluentArgumentType.Number)
-					argumentDictionary.Add(argument.Key, new FluentNumber(argument.Value));
+				if (argument.Type is FluentArgument.FluentArgumentType.Number)
+				{
+					if (!int.TryParse(argument.Value, out var number))
```

Comment:
We make a case of supporting a whole lot of number types with `TranslationExts.ToFluentType()`.
 - Is it safe to only use `int` here?
 - Do we need to check the others in `ToFluentType()` then?

Reply:
* Yes, all the arguments are integer.
* Yes, my next https://github.com/Mailaender/openra/tree/fluent-ui-logic requires other fluent types.